source/code/providers/Container_DaemonEvent_Class_Provider.cpp

```diff
@@ -18,6 +18,7 @@
 #include "Container_ContainerLogFileReader.h"
 
 #define LASTQUERYTIMEFILE "/var/opt/microsoft/docker-cimprov/state/LastEventQueryTime.txt"
+#define TEST_LASTQUERYTIMEFILE "/tmp/LastEventQueryTime.txt"

```

Comment:
So this is a problem if multiple people work on the unit tests. Why not make this something like ./LastEventQueryTime.txt, which would allow two people (working on the same machine) to not clobber each other, since they'd be in different directory trees?